### ISSUE TITLE
Talk about different goals 

### ISSUE THREAD

**Issue by zanderle** [williln/williln.github.io#9]
It might be worth talking about (or at least mentioning) the importance of finding things that the reader wants to achieve. Coding is a tool, and it's much easier to learn with a goal in mind. I think using the tools  (coding and others) on something one is passionate about, is a way to progress very quickly. It makes all those tutorials and resources get a whole new sense.

Related to that, maybe the site should state more clearly that its focus is on web development (with Python and Django) and on the very beginnings - for other topics there might be other, better sources.

**Comment by williln** [comment]
@zanderle Yeah, I agree with you. 

What do you think of having different "buttons" (at some point, this may even need to graduate to an actual menu!) with different goals? Like, in addition to "I have no prior experience," having, "I want to build websites," "I want to automate things at my desk job," I want to ..." [insert things I can't think of right now].

**Comment by zanderle** [comment]
Yeah, having different buttons sounds like a good way of solving this!
